zgw_consumers/contrib/setup_configuration/steps.py

```diff
@@ -0,0 +1,47 @@
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.exceptions import SelfTestFailed
+
+from zgw_consumers.models import Service
+
+from .models import ServicesConfigurationModel
+
+
+class ServiceConfigurationStep(BaseConfigurationStep[ServicesConfigurationModel]):
+    """
+    Configure Services to connect with external APIs
+    """
+
+    verbose_name = "Configuration to connect with external services"
+    config_model = ServicesConfigurationModel
+    namespace = "ZGW_CONSUMERS"
+    enable_setting = "ZGW_CONSUMERS_CONFIG_ENABLE"
+
+    def is_configured(self, model: ServicesConfigurationModel) -> bool:
+        slugs = [config.slug for config in model.services]
+        return Service.objects.filter(slug__in=slugs).count() == len(slugs)
```

Comment:
I'm not sure when we would consider a step as "configured". For instance as it is implemented now, it only checks if all slugs are present and not if the other values are the same as well. If one slug is missing, it will execute currently

Reply:
discussed, `is_configured` will be removed